code/script.js

```diff
@@ -0,0 +1,136 @@
+        `
+        })
+}
+
+//Function to fetch and display my repos
+const fetchRepos = () => {
+    fetch(MY_REPOS)
+        .then((res) => res.json())
+        .then((data) => {
+            projectsContainer.innerHTML = `` //Clear innerHTML to prevent duplicated repos
+            const forkedRepos =
+                data.filter(repo => {
+                    return repo.fork && repo.name.startsWith("project-") && selectedLanguages.includes(repo.language)
+                })
+            //Function to sort repos by latest date
+            forkedRepos.sort(function (a, b) {
+                return new Date(b.pushed_at) - new Date(a.pushed_at)
+            })
+            forkedRepos.forEach(repo => {
+                projectsContainer.innerHTML += `
+                      <div class="repo" id=${repo.name}>
+                        <p>Name: ${repo.name} </p> 
+                        <a href="${repo.html_url}" target="blank">Go to repo!</a> 
+                        <p>Branch: ${repo.default_branch}</p>
+                        <p>Latest push: ${new Date(repo.pushed_at).toDateString()}</p>
+                        <p id="commit-${repo.name}">Amount of commits: </p> 
+                      </div>
+                        `
+            })
+
+            fetchPullRequests(forkedRepos)
+
+            //Only draw the chart once
+            if (!chartDrawn) {
+                drawChart(forkedRepos.length)
+                chartDrawn = true
+            }
+        })
+}
+
+const fetchPullRequests = (repos) => {
+    repos.forEach(repo => {
+        const PULL_URL = `https://api.github.com/repos/Technigo/${repo.name}/pulls?per_page=100`
+
+        fetch(PULL_URL)
+            .then(res => res.json())
+            .then((data) => {
+                const myPullRequest =
+                    data.find(pull => pull.user.login === repo.owner.login)
+                if (myPullRequest) {
+                    fetchCommits(myPullRequest.commits_url, repo.name);
+                } else {
+                    document.getElementById(`commit-${repo.name}`).innerHTML =
+                        "No pull request yet!";
+                }
+            })
+    })
+}
+
+const fetchCommits = (myCommitsUrl, myRepoName) => {
+    fetch(myCommitsUrl)
+        .then(res => res.json())
+        .then(data => {
+            document.getElementById(`commit-${myRepoName}`).innerHTML += data.length
+        })
+}
+
+const filterLanguages = (event) => {
+    console.log(event)
+    const language = event.target.id
+    const checked = event.target.checked
+
+    console.log(language, checked)
+
+    //Adds language string in array if checked language is missing
+    if (checked && !selectedLanguages.includes(language)) {
+        selectedLanguages.push(language)
+        //Removes language string in array if unchecked language is in array     
+    } else if (!checked && selectedLanguages.includes(language)) {
+        for (let i = 0; i < selectedLanguages.length; i++) {
```

Comment:
Great use of for loop also cool that you are implementing other methods (not only foreach) that we learned so far.